chanzuckerberg/czid-platformics
# Expose deprecatedById in workflowRuns query interface

Updates codegen to expose ids of related objects from the same table in the query interface, i.e.:
* On `taxa`, allow users to query for `taxClassId`, `taxFamilyId`, etc.
* On `workflowRuns`, allow users to query for `deprecatedById`

Manually tested:
* Update one `workflowRun` to be `deprecatedBy` another
<img width="1375" alt="Screenshot 2024-04-18 at 4 49 57 PM" src="https://github.com/chanzuckerberg/czid-platformics/assets/53838890/0310a9b9-6dc3-47b6-82fd-6f43db9caaa6">

* Verify that the `deprecatedById` is set as expected, and we can query for it
<img width="1384" alt="Screenshot 2024-04-18 at 4 50 42 PM" src="https://github.com/chanzuckerberg/czid-platformics/assets/53838890/017ad097-128b-47ff-9093-df1f8f4e13a6">
